### PR TITLE
Update README.md with codespaces instructions

### LINKED ISSUE #61: [DOCS] Add steps to run project in codespaces
### Description

steps for running in codespaces are similar to local with docker, add notes to make clear the differences

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by amandamartin-dev**
## Fixes Issue #61 

 Closes #61 

## Changes proposed
Added the basic modifications to the setup required to use the codespaces cloud dev en provided by github.  Also added a space in a previous step to separate a/b

## Check List (Check all the applicable boxes) 

- [ X] My code follows the code style of this project.
- [ X] My change requires changes to the documentation.
- [X ] I have updated the documentation accordingly.
- [ N/A] All new and existing tests passed.
- [X ] This PR does not contain plagiarized content.
- [X ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,4 +62,18 @@ How friendly is your GitHub Open Source Repo? This project will check to make su
 9. Run the project with one of these
 
    a. If you have Postgres installed, you can run the app locally `npm run dev` OR
+   
    b. If you don't have Postgres installed, you can use Docker Compose to run the database and the app `docker compose -f docker-compose.yaml up -d`, after the containers are ready, run the command `npm run db:migrate:dev`
+
+10. Running in Github Codespaces
```
Comment (eddiejaoude) [inline README.md:68]: Thanks Amanda for figuring this out!

Should this be `c` (9c)?

Then `a` becomes nested further?

Comment (amandamartin-dev) [reply README.md:68]: Oh, sure that works too! Updating now

File: README.md [hunk 2]
```diff
@@ -62,4 +62,13 @@ How friendly is your GitHub Open Source Repo? This project will check to make su
 9. Run the project with one of these
 
    a. If you have Postgres installed, you can run the app locally `npm run dev` OR
+   
    b. If you don't have Postgres installed, you can use Docker Compose to run the database and the app `docker compose -f docker-compose.yaml up -d`, after the containers are ready, run the command `npm run db:migrate:dev`
+
+   c. Running in Github Codespaces
+      1. Start a new codespace
+      2. Run `npm install` and `npm run dev`
```
Comment (eddiejaoude) [inline README.md:70]:
```suggestion
      2. Run `npm ci` and `npm run dev`
```

File: README.md [hunk 3]
```diff
@@ -62,4 +62,13 @@ How friendly is your GitHub Open Source Repo? This project will check to make su
...
    a. If you have Postgres installed, you can run the app locally `npm run dev` OR
+   
    b. If you don't have Postgres installed, you can use Docker Compose to run the database and the app `docker compose -f docker-compose.yaml up -d`, after the containers are ready, run the command `npm run db:migrate:dev`
+
+   c. Running in Github Codespaces
+      1. Start a new codespace
+      2. Run `npm ci` and `npm run dev`
+      3. When the project is running in the browser visit this and copy the generated URL. You will use this URL in place of localhost in step 8 above for creating your OAuth app
+      4. Update the environment file with your client id and secret.  Make sure to also update the `NEXTAUTH_URL=` to the generated codespace URL as well.
+      5. Continue with step 9b to run the docker container and database migration.
```
Comment (eddiejaoude) [inline README.md:73]:
```suggestion
      5. Continue with step `9b` to run the docker container and database migration.
```

**Review by eddiejaoude — APPROVED** [review]
Awesome! Thank you

**Pull request merged**